modules/ROOT/pages/_partials/prometheus-activation.adoc

```diff
@@ -0,0 +1,18 @@
+// BEGIN -- inclusion -- prometheus-activation.adoc
+//
+To activate the Metrics REST API you need to edit your `sync-gateway-config.json` configuration file.
+
+Include the `metricsInterface` setting and define the Sync Gateway URL and the port you want the API served on (default: 4986) -- see: <<ex-activate>>.
```

Comment:
4986 is not the default really. Unless specified the metrics endpoint is not accessible like you mention in the first sentence.

Reply:
Amended, thanks.